addons/osfstorage/tests/test_views.py

```diff
@@ -942,7 +971,26 @@ def test_attempt_delete_double_nested_folder_rented_file(self):
 
 
 @pytest.mark.django_db
-class TestDeleteHookPreprint(TestDeleteHook):
+class TestDeleteHookProjectOnly(DeleteHook):
+
+    def test_delete_reduces_cache_size(self):
```

Comment:
Some awkward changes `TestDeleteHookProjectOnly` with here because `test_delete_reduces_cache_size` is the only test in the suite that __only__ works for projects and not preprints.